include/fastdds/statistics/topic_names.hpp

```diff
@@ -0,0 +1,69 @@
+// Copyright 2021 Proyectos y Sistemas de Mantenimiento SL (eProsima).
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @file topic_names.hpp
+ */
+
+#ifndef _STATISTICS_TOPIC_NAMES_HPP_
+#define _STATISTICS_TOPIC_NAMES_HPP_
+
+namespace eprosima {
+namespace fastdds {
+namespace dds {
+namespace statistics {
+
+//! Statistic topic that reports the write-to-notification latency between any two pairs of matched
+//! DataWriter-DataReader histories
+constexpr char HISTORY_LATENCY_TOPIC[] = "_fastdds_statistics_history2history_latency";
```

Comment:
This suggestion applies to all the constants on this file

```suggestion
constexpr const char* HISTORY_LATENCY_TOPIC = "_fastdds_statistics_history2history_latency";
```